Anguandia/quickcredit
# Ft db view unapproved loans

#### Purpose of this pull request

Add query parameter and parameter value validation

#### Tasks To be accomplished

- Validate that GET/loans route parameters are either status or repaid
- Validate the allowed values for these two parameters from the query string if present

How to manually test request query validation validation

- Run any installation of this app, local or hosted, with or without loans in the database
- Send a Get/loans?query request with query parameters other than status and repaid
- In another turn, send a similar request with the valid parameters but wrong values and observe the response

Comments

- Nothing anyway related to database integration

Pivotal tracker stories

[#166229490 #166115468 #166115442]
